src/Features/Core/Portable/ExternalAccess/UnitTesting/UnitTestingSolutionCrawlerServiceAccessor.cs

```diff
@@ -12,13 +12,38 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting
 {
     internal sealed class UnitTestingSolutionCrawlerServiceAccessor : IUnitTestingSolutionCrawlerServiceAccessor
     {
-        private readonly ISolutionCrawlerService _implementation;
+        private readonly ISolutionCrawlerRegistrationService _registrationService;
+        private readonly ISolutionCrawlerService _solutionCrawlerService;
 
-        [Obsolete(MefConstruction.FactoryMethodMessage, error: true)]
-        public UnitTestingSolutionCrawlerServiceAccessor(ISolutionCrawlerService implementation)
-            => _implementation = implementation;
+        private UnitTestingIncrementalAnalyzerProvider _analyzerProvider;
 
-        public void Reanalyze(Workspace workspace, IUnitTestingIncrementalAnalyzerImplementation analyzer, IEnumerable<ProjectId> projectIds = null, IEnumerable<DocumentId> documentIds = null, bool highPriority = false)
-            => _implementation.Reanalyze(workspace, new UnitTestingIncrementalAnalyzer(analyzer), projectIds, documentIds, highPriority);
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
```

Comment:
This changed from FactoryMethodMessage to ImprotingConstructorMessage. What does this change do?

Reply:
not sure which one is right, it's actually taken from UnitTestingSolutionCrawlerRegistrationServiceAccessor which I renamed. @Cosifne might have more insight into this

Reply:
Looks like this should be changed back to FactoryMethodMessage since this is constructed via a factory...

Reply:
I also think it should FactoryMethodMessage